item-spec/item-spec.md

```diff
@@ -66,8 +66,10 @@ Items that are linked to, but the best practices around this are still emerging.
 
 ### Properties Object
 
-The Properties object adds additional metadata to the GeoJSON Object. Additional fields can be introduced through 
-extensions. It is generally allowed to add custom fields.
+The Properties object adds additional metadata to the GeoJSON Object. The Item definition also supports a default suite 
+of optional [Common Metadata](common-metadata.md) fields to describe information such as metadata update datetimes, licensing,
+provider attribution, etc. Additional domain-specific fields can be introduced through [extensions](../extensions/README.md#extensions). 
+It is generally allowed to add custom fields.
 
 It is recommended to add multiple attributes for related values instead of a nested object, e.g., two fields `eo:cloud_cover` and `sat:sun_azimuth_angle` instead of a field `eo` with an object value containing the two fields. The convention (as used within Extensions) is for related attributes to use a common prefix on the attribute names to group them, e.g. `eo`. A nested data structure should only be used when the data itself is nested, as with `eo:bands`.
```

Comment:
I think that everything starting with It is generally allowed to add custom fields is no longer needed. This is explained in the Extensions page.

Reply:
I don't get this? I think it's still important to mention that custom fields are allowed.

Reply:
I think the text I added makes the transition awkward is what was bothering me. 

What do you think about the next paragraph that explains the details of how extensions should be defined, such as nesting. Does that stay as well? It seems like we need a quick statement that says users can define custom fields, see the extensions page extensions#extending-stac for more information.

Reply:
Yeah, at the moment it's not optimal, also the bold text below the table duplicates what is said above. What about replacing the chapter with:

### Properties Object

The Properties object allows to add additional metadata fields to the GeoJSON Object. The only required field is `datetime`:

| Field Name | Type   | Description                                                  |
| ---------- | ------ | ------------------------------------------------------------ |
| datetime   | string | **REQUIRED.** The searchable date and time of the assets, in UTC. It is formatted according to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6). |
| title      | string | A human readable title describing the item. |

In addition, it is recommended to add more suitable fields from any of the sources below:
* **[STAC Common Metadata](common-metadata.md)**: A list of commonly used fields throughout all domains
* **[Content Extensions](../extensions/README.md#list-of-content-extensions)**: Domain-specific fields for domains such as EO, SAR and point clouds.
* **Custom Properties**: It is generally allowed to add custom fields, but it is recommended to add multiple attributes for related values instead of a nested object, e.g., two fields `eo:cloud_cover` and `sat:sun_azimuth_angle` instead of a field `eo` with an object value containing the two fields. The convention (as used within Extensions) is for related attributes to use a common prefix on the attribute names to group them, e.g. `eo`. A nested data structure should only be used when the data itself is nested, as with `eo:bands`.

Reply:
@anayeaye I saw your "thumbs up". Could you commit the changes to this PR? I can't commit to your branch :-)

Reply:
Yup. Wanted to let you know I saw it. I can get the changes in later today or tomorrow.

Reply:
Ah, great! Understood it as "I agree to the proposal, please change it" ;-)

Reply:
Thanks for the changes, @anayeaye . You also added the JSON Schema, but forgot to remove the schema for title from the item.json schema.